keel-api/src/test/resources/examples/titus-cluster-example.yml

```diff
@@ -0,0 +1,51 @@
+---
+apiVersion: titus.spinnaker.netflix.com/v1
+kind: titus-cluster
+metadata:
+  serviceAccount: my-email@spinnaker.io
+spec:
+  moniker:
+    app: keeldemo
+    stack: examples
+    detail: ec2v1
+  container:
+    registry: testregistry
+    organization: emburns
+    image: emburns/spin-titus-demo
+    digest: ""
+    tag: latest
+  locations:
+    account: titustestvpc
+    regions:
+      - name: us-west-2
+      - name: us-east-1
+  containerOptions:
```

Comment:
Minor nit but these properties tend to configure the runtime environment so I don't love naming this block `containerOptions`.

Reply:
I think in a future PR I'll experiment with pulling all these up to top level things (and defaulting most of them)